addons/mail/models/discuss/discuss_channel.py

```diff
@@ -907,7 +905,7 @@ def _channel_info(self):
                     info['seen_message_id'] = member.seen_message_id.id
                     info['custom_channel_name'] = member.custom_channel_name
                     info['is_pinned'] = member.is_pinned
-                    info['last_interest_dt'] = fields.Datetime.to_string(member.last_interest_dt)
+                    info['last_interest_dt'] = fields.Datetime.to_string(member.last_interest_dt if not channel.last_interest_dt or member.last_interest_dt > channel.last_interest_dt else channel.last_interest_dt)
```

Comment:
This shouldn't be changed, we should have 2 separate value in code for channel dt and member dt.

Reply:
If there is no member, we still want to return the value for channel last_interest_dt too.

Reply:
why do we want that when there is no member? the old logic was only having the `last_interest_dt` when there is a member and so that we have `last_interest_dt` only in `thread` model on the client side. When we having both, then we might need to do the compute in the client side once any one of it is getting updated since we only want to have one `last_interest_dt` in the `thread` model

Reply:
We can display channels while not being member (livechat history is one way) so we need to handle the case.
We still have usage of `last_interest_dt` to sort those threads etc.

So we need both in JS, with proper compute as well.